CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -273,11 +273,15 @@ end
 
 function Methods.ResetFrameDimensions(devConsole)
 	devConsole.Frame.Size = UDim2_new(0.5, 20, 0.5, 20);
-	devConsole.Frame.Position = UDim2_new(0.25, -10, 0.125, -10)
+	
+	local abSize = devConsole.Frame.AbsoluteSize
+	devConsole:SetFrameSize(abSize.x, abSize.y)
```

Comment:
This is a check to make sure that when the developer console first appears, it isn't smaller than what is defined as the minimum size. I tried to keep it so that it wouldn't mess with its original behavior. I could rewrite some of it to make it more elegant.